common/persistence/nosql/nosqlplugin/cassandra/workflow_utils_test.go

```diff
@@ -259,35 +320,70 @@ func TestExecuteCreateWorkflowBatchTransaction(t *testing.T) {
 			},
 		},
 		{
-			desc: "execution already exists",
+			desc: "workflow request already exists",
 			session: &fakeSession{
 				mapExecuteBatchCASApplied: false,
 				iter:                      &fakeIter{},
 				mapExecuteBatchCASPrev: map[string]any{
-					"type":                        rowTypeExecution,
-					"run_id":                      uuid.Parse(permanentRunID),
-					"range_id":                    int64(100),
-					"workflow_last_write_version": int64(3),
-					"execution": map[string]any{
-						"workflow_id": "test-workflow-id",
-						"run_id":      uuid.Parse("bda9cd9c-32fb-4267-b120-346e5351fc46"),
-						"state":       1,
+					"type":   rowTypeWorkflowRequestStart,
+					"run_id": uuid.Parse("4b8045c8-7b45-41e0-bf03-1f0d166b818d"),
+				},
+				query: &fakeQuery{
+					mapScan: map[string]interface{}{
+						"current_run_id": uuid.Parse("6b844fb4-c18a-4979-a2d3-731ebdd1db08"),
 					},
 				},
 			},
 			batch:        &fakeBatch{},
 			currentWFReq: &nosqlplugin.CurrentWorkflowWriteRequest{},
+			execution: &nosqlplugin.WorkflowExecutionRequest{
+				InternalWorkflowExecutionInfo: persistence.InternalWorkflowExecutionInfo{
+					DomainID:   "fd88863f-bb32-4daa-8878-49e08b91545e",
+					WorkflowID: "wfid",
+				},
+			},
 			shardCond: &nosqlplugin.ShardCondition{
+				ShardID: 1,
 				RangeID: 100,
 			},
 			wantErr: &nosqlplugin.WorkflowOperationConditionFailure{
-				CurrentWorkflowConditionFailInfo: common.StringPtr(
```

Comment:
why removing this case? is it covered by something else? If possible let's leave existing test cases intact to avoid changing behavior

Reply:
it's renamed to `current execution already exists and write mode is CurrentWorkflowWriteModeInsert`